packages/teams-js/package.json

```diff
@@ -5,7 +5,8 @@
   "description": "Microsoft Client SDK for building app for Microsoft hosts",
   "repository": {
     "type": "git",
-    "url": "https://github.com/OfficeDev/microsoft-teams-library-js"
+    "url": "https://github.com/OfficeDev/microsoft-teams-library-js",
+    "directory": "packages/teams-js"
```

Comment:
>"directory": "packages/teams-js" [](http://example.com/codeflow?start=4&length=32)

Should these be alphabetized? `directory` before `type`?  <shrug>

Reply:
I'm happy to, though npm docs has it in this order, as does the Fluid Framework (which I used to confirm usage of another well-known monorepo).